easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.1.0-foss-2019b-Python-3.7.4.eb

```diff
@@ -35,15 +53,14 @@ exts_list = [
     ('scipy', '1.4.1', {
```

Comment:
@Flamefire Should be removed and replaced with a proper `scipy` dep just like in the `fosscuda` easyconfig?